app/models/status_message.rb

```diff
@@ -149,10 +149,5 @@ def absence_of_content
   def self.tag_stream(tag_ids)
     joins(:taggings).where('taggings.tag_id IN (?)', tag_ids)
   end
-
-  def after_parse
```

Comment:
That what I thought.

`rgrep _parse app` gives nothing. But I guess with the coverage tool searching for unused code is an easy walk :)